README.md

```diff
@@ -1,19 +1,33 @@
 # SnarkyJS &nbsp; [![npm version](https://img.shields.io/npm/v/snarkyjs.svg?style=flat)](https://www.npmjs.com/package/snarkyjs) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md)
 
-The easiest way to write zk programs is using SnarkyJS, a TypeScript framework for zk-SNARKs and zkApps.
+The SnarkyJS SDK helps developers build apps powered by zero-knowledge cryptography.
 
-To write a zkApp smart contract for the Mina Protocol, we recommend using the [zkApp CLI](https://github.com/o1-labs/zkapp-cli). The zkApp CLI allows you to scaffold, write, test, and deploy zkApps using recommended best practices. zkApps created with the zkApp CLI include SnarkyJS and provide project scaffolding, a test framework, and correct formatting.
+SnarkyJS is a TypeScript library for writing smart contracts based on zero-knowledge proofs for the Mina Protocol. The easiest way to write zero-knowledge (zk) programs is using the SnarkyJS TypeScript framework for zk-SNARKs and zkApps. 
```

Comment:
> SnarkyJS is a TypeScript library for writing smart contracts based on zero-knowledge proofs for the Mina Protocol.

I think this is a bit limiting. SnarkyJS is two things:
* a general-purpose zk framework that can be used for all kinds of zk apps
* a zk smart contract framework tailored to Mina

Your sentence makes it sounds like only the second of these

> The easiest way to write zero-knowledge (zk) programs is using the SnarkyJS TypeScript framework for zk-SNARKs and zkApps. 

This sounds a bit redundant IMO, will think of what we could add here